lighthouse-core/computed/metrics/timing-summary.js

```diff
@@ -0,0 +1,106 @@
+    const totalBlockingTime = await TotalBlockingTime.request(metricComputationData, context); // eslint-disable-line max-len
+
+    /** @type {LH.Artifacts.TimingSummary} */
+    const metrics = {
+      // Include the simulated/observed performance metrics
+      firstContentfulPaint: firstContentfulPaint.timing,
+      firstContentfulPaintTs: firstContentfulPaint.timestamp,
+      firstMeaningfulPaint: firstMeaningfulPaint.timing,
+      firstMeaningfulPaintTs: firstMeaningfulPaint.timestamp,
+      largestContentfulPaint: largestContentfulPaint && largestContentfulPaint.timing,
+      largestContentfulPaintTs: largestContentfulPaint && largestContentfulPaint.timestamp,
+      firstCPUIdle: firstCPUIdle && firstCPUIdle.timing,
+      firstCPUIdleTs: firstCPUIdle && firstCPUIdle.timestamp,
+      interactive: interactive && interactive.timing,
+      interactiveTs: interactive && interactive.timestamp,
+      speedIndex: speedIndex && speedIndex.timing,
+      speedIndexTs: speedIndex && speedIndex.timestamp,
+      estimatedInputLatency: estimatedInputLatency.timing,
+      estimatedInputLatencyTs: estimatedInputLatency.timestamp,
+      totalBlockingTime: totalBlockingTime.timing,
+
+      // Include all timestamps of interest from trace of tab
+      observedNavigationStart: traceOfTab.timings.navigationStart,
+      observedNavigationStartTs: traceOfTab.timestamps.navigationStart,
+      observedFirstPaint: traceOfTab.timings.firstPaint,
+      observedFirstPaintTs: traceOfTab.timestamps.firstPaint,
+      observedFirstContentfulPaint: traceOfTab.timings.firstContentfulPaint,
+      observedFirstContentfulPaintTs: traceOfTab.timestamps.firstContentfulPaint,
+      observedFirstMeaningfulPaint: traceOfTab.timings.firstMeaningfulPaint,
+      observedFirstMeaningfulPaintTs: traceOfTab.timestamps.firstMeaningfulPaint,
+      observedLargestContentfulPaint: traceOfTab.timings.largestContentfulPaint,
+      observedLargestContentfulPaintTs: traceOfTab.timestamps.largestContentfulPaint,
+      observedTraceEnd: traceOfTab.timings.traceEnd,
+      observedTraceEndTs: traceOfTab.timestamps.traceEnd,
+      observedLoad: traceOfTab.timings.load,
+      observedLoadTs: traceOfTab.timestamps.load,
+      observedDomContentLoaded: traceOfTab.timings.domContentLoaded,
+      observedDomContentLoadedTs: traceOfTab.timestamps.domContentLoaded,
+
+      // Include some visual metrics from speedline
+      observedFirstVisualChange: speedline.first,
+      observedFirstVisualChangeTs: (speedline.first + speedline.beginning) * 1000,
+      observedLastVisualChange: speedline.complete,
+      observedLastVisualChangeTs: (speedline.complete + speedline.beginning) * 1000,
+      observedSpeedIndex: speedline.speedIndex,
+      observedSpeedIndexTs: (speedline.speedIndex + speedline.beginning) * 1000,
+    };
+    /** @type {Array<Record<string,boolean>>} */
+    const debugInfo = [{lcpInvalidated: trace.lcpInvalidated}];
```

Comment:
this isn't the right value, I'm surprised that ts lets this happen. I think we want `traceOfTab.lcpInvalidated`?